### DISCUSSION TITLE
relation does not exist with trigger

### DISCUSSION THREAD

**Discussion by Samsonov0** [supabase/supabase#12398] [Questions]
# Bug report

## Describe the bug

I have a table extending auth.users by o2o key. In the table there is a field calling a custom function to get a random number, and in this function there is an enumerate to get a sequence of numbers. I wanted to create a trigger to automatically create a user profile during registration and began to face the problem "Failed to invite user: failed to make invite request: Database error saving new user". After looking at the logs, I saw that when sending a request, he complains about my enumerate that it does not exist
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create enum
2. Create a function that will call this enumerate
3. Set this function to the default value in the table field
4. Create a trigger to add an entry to the profile table during registration
5. Try inviting a user by mail
6. See the problem

## Expected behavior

I will be able to invite a user

## Screenshots

![image](https://user-images.githubusercontent.com/52374669/218036462-b56d481e-fe94-45fa-88b7-1f3126653eaf.png)
![image](https://user-images.githubusercontent.com/52374669/218036491-9bfc3b2b-97f1-42b0-bca5-764baba9ceda.png)
![image](https://user-images.githubusercontent.com/52374669/218036505-d1b3cf0f-fb34-40ca-8e91-364a239fbc21.png)

**Comment by GaryAustin1** [comment]
You’ll will need to provide your function for anyone to help.  Seems there is an error finding your function. May be as simple as adding public in front of it or a search path.

**Comment by Samsonov0** [comment]
> You’ll will need to provide your function for anyone to help. Seems there is an error finding your function. May be as simple as adding public. In front of it or a search path.

thank you for the advice with the indication of the function, I will definitely leave it. Regarding the indication of the path, I have already tried. To no avail

**Comment by GaryAustin1** [comment]
I'm going to transfer this issue to a discussion.  Auth.users table trigger functions can call other functions.  Once you are in the trigger function, if you are not using auth.user specific columns, then you should be using pure Postgres so likely not a bug in Supabase.

Please feel free to provide more info so it can be narrowed down as to what the issue is.
Thanks
